pkgs/top-level/all-packages.nix

```
@@ -26495,6 +26495,8 @@ with pkgs;
   };
 
   git = callPackage ../applications/version-management/git-and-tools/git {
+    inherit (darwin) Security;
+    inherit (darwin.apple_sdk.frameworks) CoreServices;
```

The top-level `darwin package set just splices some of the frameworks into it, I think this is a little more clean
```suggestion
    inherit (darwin.apple_sdk.frameworks) CoreServices Security;
```

---

Thanks, I merged the two statements.

I tried doing it via the web interface, but Github wasn’t cooperating, so I updated and force-pushed a clean commit.